tests/next_tests/iterator_tests/test_with_toy_connectivity.py

```diff
@@ -214,30 +261,30 @@ def slice_twice_sparse_stencil(sparse):
     return deref(shift(2)(shift(1)(sparse)))
 
 
-def test_slice_twice_sparse(program_processor_no_gtfn_exec, lift_mode):
-    program_processor, validate = program_processor_no_gtfn_exec
-    inp = np_as_located_field(Vertex, V2V, V2V)(v2v_arr[v2v_arr])
-    out = np_as_located_field(Vertex)(np.zeros([9]))
+# def test_slice_twice_sparse(program_processor_no_gtfn_exec, lift_mode):
```

Comment:
TODO implement double shifted field in embedded.
Is it `It[...,List[List[dtype]]` or `It[..., List[It[List[dtype]]]]`?